snippets/brighterscript.json

```diff
@@ -0,0 +1,44 @@
+{
+    "Class": {
+        "prefix": "class",
+        "body": [
+            "class ${1:ClassName}",
+            "\tsub new()",
```

Comment:
Humm is there a way to provide this matching the user/workspace setting for indentation? Or with VS Code already handle this when injecting the snippet?

Reply:
`\t` in the snippet body matches whatever your workspace indentation settings are.